web/src/css/6-components/_know-your-rights-route.scss

```diff
@@ -0,0 +1,21 @@
+.RightsList {
+    font-family: Times;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    padding: 0.6rem;
+}
+
+.RightsItem h6 {
+    font-weight: bold;
+    font-size: 1.2em;
+    padding: 0.6rem;
+    display: block;
+
+}
+
+.RightsItem li {
+    padding-left: 1rem;
```

Comment:
You can consolidate `padding-left` and `padding-right` into a shorthand for padding. Specifically `padding: 0 1rem;`.

https://www.w3schools.com/css/css_padding.asp